framework/src/testing/app_env.ts

```diff
@@ -123,3 +131,60 @@ export class ApplicationEnv {
 		return application;
 	}
 }
+
+export const createDefaultApplicationEnv = (
+	appEnvConfig: Partial<ApplicationEnvConfig>,
```

Comment:
i think extra module should be addable as optional here, otherwise it's not useful to others